Successfully updated:
Created Frame node WNlgVgHBF
Created Frame node BWt9tOxUe
Created Text node YDhvCY_jz
Created Text node vJQkz_5k7
Created Frame node G_WP_pdPP
Updated attributes for node WNlgVgHBF
Updated attributes for node BWt9tOxUe
Updated text for node YDhvCY_jz
Updated attributes for node YDhvCY_jz
Updated text for node vJQkz_5k7
Updated attributes for node vJQkz_5k7
Updated attributes for node G_WP_pdPP
Reordered node BWt9tOxUe within parent WNlgVgHBF to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -3871,20 +3871,73 @@
     top="0px"
     left="0px"
     <!-- background color string or project color style path (if starts with /) -->
     backgroundColor="rgb(200, 200, 200)"
     zIndex="10"
     overflow="hidden"
 >
   <TruncatedTextThatIsLongEnoughToBeTruncated
       nodeId="c5HU2SqaR"
       position="absolute"
       width="fit-content"
       height="fit-content"
       top="0px"
       left="0px"
       font="Inter"
       textTruncation="2"
   >
     Truncated Text that is long enough to be truncated
   </TruncatedTextThatIsLongEnoughToBeTruncated>
 </Frame>
+<Stack
+    nodeId="WNlgVgHBF"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="BWt9tOxUe"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="YDhvCY_jz"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="vJQkz_5k7"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="G_WP_pdPP"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.